package.json

```diff
@@ -0,0 +1,346 @@
+    "postcss-sass": "^0.4.4",
+    "postcss-scss": "^2.1.1",
+    "postcss-selector-parser": "^6.0.11",
+    "postcss-syntax": "^0.36.2",
+    "postcss-value-parser": "^4.2.0",
+    "prelude-ls": "^1.2.1",
+    "progress": "^2.0.3",
+    "punycode": "^2.2.0",
+    "queue-microtask": "^1.2.3",
+    "quick-lru": "^4.0.1",
+    "read-pkg": "^5.2.0",
+    "read-pkg-up": "^7.0.1",
+    "readable-stream": "^3.6.0",
+    "redent": "^3.0.0",
+    "regexp.prototype.flags": "^1.4.3",
+    "regexpp": "^3.2.0",
+    "remark": "^13.0.0",
+    "remark-parse": "^9.0.0",
+    "remark-stringify": "^9.0.1",
+    "repeat-string": "^1.6.1",
+    "require-from-string": "^2.0.2",
+    "resolve": "^1.22.1",
+    "resolve-from": "^5.0.0",
+    "reusify": "^1.0.4",
+    "rimraf": "^3.0.2",
+    "run-parallel": "^1.2.0",
+    "safe-buffer": "^5.2.1",
+    "safe-regex-test": "^1.0.0",
+    "semver": "^6.3.0",
+    "shebang-command": "^2.0.0",
+    "shebang-regex": "^3.0.0",
+    "side-channel": "^1.0.4",
+    "signal-exit": "^3.0.7",
+    "slash": "^3.0.0",
+    "slice-ansi": "^4.0.0",
+    "source-map": "^0.6.1",
+    "spdx-correct": "^3.1.1",
+    "spdx-exceptions": "^2.3.0",
+    "spdx-expression-parse": "^3.0.1",
+    "spdx-license-ids": "^3.0.12",
+    "specificity": "^0.4.1",
+    "sprintf-js": "^1.0.3",
+    "string_decoder": "^1.3.0",
+    "string-width": "^4.2.3",
+    "string.prototype.trimend": "^1.0.6",
+    "string.prototype.trimstart": "^1.0.6",
+    "strip-ansi": "^6.0.1",
+    "strip-bom": "^3.0.0",
+    "strip-indent": "^3.0.0",
+    "strip-json-comments": "^3.1.1",
+    "style-search": "^0.1.0",
+    "stylelint-config-recommended": "^4.0.0",
+    "sugarss": "^2.0.0",
+    "supports-color": "^7.2.0",
+    "supports-preserve-symlinks-flag": "^1.0.0",
+    "svg-tags": "^1.0.0",
+    "table": "^6.8.1",
+    "text-table": "^0.2.0",
+    "to-fast-properties": "^2.0.0",
+    "to-regex-range": "^5.0.1",
+    "trim-newlines": "^3.0.1",
+    "trough": "^1.0.5",
+    "tsconfig-paths": "^3.14.1",
+    "type-check": "^0.4.0",
+    "type-fest": "^0.18.1",
+    "typed-array-length": "^1.0.4",
+    "typedarray-to-buffer": "^3.1.5",
+    "unbox-primitive": "^1.0.2",
+    "unified": "^9.2.2",
+    "unist-util-find-all-after": "^3.0.2",
+    "unist-util-is": "^4.1.0",
+    "unist-util-stringify-position": "^2.0.3",
+    "update-browserslist-db": "^1.0.10",
+    "uri-js": "^4.4.1",
+    "util-deprecate": "^1.0.2",
+    "v8-compile-cache": "^2.3.0",
+    "validate-npm-package-license": "^3.0.4",
+    "vfile": "^4.2.1",
+    "vfile-message": "^2.0.4",
+    "which": "^1.3.1",
+    "which-boxed-primitive": "^1.0.2",
+    "which-typed-array": "^1.1.9",
+    "word-wrap": "^1.2.3",
+    "wrappy": "^1.0.2",
+    "write-file-atomic": "^3.0.3",
+    "yallist": "^3.1.1",
+    "yaml": "^1.10.2",
+    "yargs-parser": "^20.2.9",
+    "zwitch": "^1.0.5"
+  },
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "webpack"
```

Comment:
* After running the build command I get a blank screen when I open the /dist/index.html file on the browser, Kindly fix this and also add more webpack script with setup instructions on your readme file on how to run your app
![image](https://user-images.githubusercontent.com/52497246/213357163-6b47c542-48d5-4811-a8d6-2f59cef80b5e.png)
![image](https://user-images.githubusercontent.com/52497246/213357176-f444161e-0651-4c94-b1c3-cf478e59c288.png)

Reply:
Hi @Davidosky007!
I hope you are doing great. I have tried myself but unable to fix this. I'm unable to know that how to push my src html file to dist file. If you have some free time than please help me to figure out the solution. Thanks.